vttools/wrap_lib.py

```diff
@@ -668,7 +668,7 @@ def compute(self):
         ret = library_func(**params_dict)
         if len(output_ports) == 1:
             self.set_output(output_ports[0].name, ret)
-        else:
+        elif ret:
```

Comment:
should this be a check if there are _no_ output ports?  I am a tad concerned that this will hide the case where the scraper thinks that we should have output ports, but the library function does not return anything.